docs/advanced/api.mdx

```diff
@@ -130,13 +130,15 @@ const visit = require('unist-util-visit')
 const remove = require('unist-util-remove')
 const yaml = require('yaml')
 
-const file = vfile(`
+const file = vfile(
+  `
 ---
 title: Hello, MDX
 ---
 
-I <3 Markdown and JSX
-`)
+I &lt;3 Markdown and JSX {/* < must be escaped for \`@mdx-js/mdx@2\` */}
+`.trimStart() // `trimStart` is required for `remark-frontmatter@3`
+)
```

Comment:
trimStart isn't specifically required, just a lack of spaces before `---`
```js
const file = vfile(`---
title: Hello, MDX
```
would also work

Reply:
That’s the meaning to use `trimStart()`.

Reply:
Yes, that's my point, the comment

```js
// `trimStart` is required for `remark-frontmatter@3`
```

says that `trimStart` is required, it isn't.

Something like:

```js
// there should be no spaces before frontmatter, this can be guaranteed with `trimStart`